### ISSUE TITLE
jellyfin-ffmpeg invoked with invalid flags for DVD ISOs

### ISSUE THREAD

**Issue by strugee** [jellyfin/jellyfin#5615]
**Describe the bug**

When I try to play an ISO file in the web interface, Jellyfin runs the following ffmpeg command for transcode:

    /usr/lib/jellyfin-ffmpeg/ffmpeg -f iso -i bluray:"/media/Movies/THE_ACCOUNTANT.iso" -map_metadata -1 -map_chapters -1 -threads 0 -sn   -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_base_url "hls/bee06ea14e65ef9b0f374ecd049c225c/" -hls_playlist_type event -hls_segment_filename "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c%d.ts" -y "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c.m3u8"

but after spitting out some standard boilerplate output, ffmpeg reports:

    Unknown input format: 'iso'

`/usr/lib/jellyfin-ffmpeg/ffmpeg -formats 2>&1 | grep -i iso` confirms that the binary is missing ISO support.

**System (please complete the following information):**
 - OS: Debian 10
 - Virtualization: LXD
 - Clients: browser
 - Browser: Firefox 78.9 ESR
 - Jellyfin Version: 10.7.1
 - Playback: transcode (presumably)
 - Installed Plugins: AudioDB, Bookshelf, Kodi Sync Queue, MusicBrans, OMDb, Playback Reporting, Reports, TheTVDB, Trakt
 - Reverse Proxy: Apache
 - Base URL: none
 - Networking: LXD bridge
 - Storage: local (it's a LXD mount from the host filesystem)

**To Reproduce**

1. Play a movie in ISO format in your library
2. Observe that the movie never starts and Jellyfin's UI hangs
3. Check the logs

**Expected behavior**

ffmpeg was able to transcode the movie to be played in the browser.

**Logs**

The relevant section of the Jellyfin log (I suspect that the ffprobe exception is also a bug and can file a followup if so):

```
[2021-03-24 23:43:10.330 +00:00] [WRN] Slow HTTP Response from "http://jellyfin.strugee.net/Items/768d7dbf900a55af4c469fee2fd8140e/Images/Primary?maxWidth=223&tag=d17eaf37c871a71c32e9bcb9ea87e929&quality=90" to "10.133.196.1" in 0:00:00.5296282 with Status Code 200
[2021-03-24 23:43:10.399 +00:00] [WRN] Slow HTTP Response from "http://jellyfin.strugee.net/Items/e5bf8b3b4c4db610f78ab17f285f6d39/Images/Primary?maxWidth=223&tag=c568e2c45c14e29b9a5d06ee5a538a34&quality=90" to "10.133.196.1" in 0:00:00.5899716 with Status Code 200
[2021-03-24 23:43:35.644 +00:00] [INF] GetPostedPlaybackInfo profile: DeviceProfile { Name: null, Id: null, Identification: null, FriendlyName: null, Manufacturer: null, ManufacturerUrl: null, ModelName: null, ModelDescription: null, ModelNumber: null, ModelUrl: null, SerialNumber: null, EnableAlbumArtInDidl: False, EnableSingleAlbumArtLimit: False, EnableSingleSubtitleLimit: False, SupportedMediaTypes: "Audio,Photo,Video", UserId: null, AlbumArtPn: null, MaxAlbumArtWidth: 0, MaxAlbumArtHeight: 0, MaxIconWidth: null, MaxIconHeight: null, MaxStreamingBitrate: 120000000, MaxStaticBitrate: 100000000, MusicStreamingTranscodingBitrate: 384000, MaxStaticMusicBitrate: null, SonyAggregationFlags: null, ProtocolInfo: null, TimelineOffsetSeconds: 0, RequiresPlainVideoItems: False, RequiresPlainFolders: False, EnableMSMediaReceiverRegistrar: False, IgnoreTranscodeByteRangeRequests: False, XmlRootAttributes: [], DirectPlayProfiles: [DirectPlayProfile { Container: "webm", AudioCodec: "vorbis,opus", VideoCodec: "vp8,vp9,av1", Type: Video }, DirectPlayProfile { Container: "mp4,m4v", AudioCodec: "aac,mp3,opus,flac,vorbis", VideoCodec: "h264,vp8,vp9,av1", Type: Video }, DirectPlayProfile { Container: "opus", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "webm", AudioCodec: "opus", VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "mp3", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "aac", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "m4a", AudioCodec: "aac", VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "m4b", AudioCodec: "aac", VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "flac", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "webma", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "webm", AudioCodec: "webma", VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "wav", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "ogg", AudioCodec: null, VideoCodec: null, Type: Audio }], TranscodingProfiles: [TranscodingProfile { Container: "ts", Type: Audio, VideoCodec: null, AudioCodec: "aac", Protocol: "hls", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 1, SegmentLength: 0, BreakOnNonKeyFrames: True }, TranscodingProfile { Container: "aac", Type: Audio, VideoCodec: null, AudioCodec: "aac", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp3", Type: Audio, VideoCodec: null, AudioCodec: "mp3", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "opus", Type: Audio, VideoCodec: null, AudioCodec: "opus", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "wav", Type: Audio, VideoCodec: null, AudioCodec: "wav", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "opus", Type: Audio, VideoCodec: null, AudioCodec: "opus", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp3", Type: Audio, VideoCodec: null, AudioCodec: "mp3", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "aac", Type: Audio, VideoCodec: null, AudioCodec: "aac", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "wav", Type: Audio, VideoCodec: null, AudioCodec: "wav", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "ts", Type: Video, VideoCodec: "h264", AudioCodec: "aac,mp3", Protocol: "hls", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 1, SegmentLength: 0, BreakOnNonKeyFrames: True }, TranscodingProfile { Container: "webm", Type: Video, VideoCodec: "vpx", AudioCodec: "vorbis", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp4", Type: Video, VideoCodec: "h264", AudioCodec: "aac,mp3,opus,flac,vorbis", Protocol: "http", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Static, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }], ContainerProfiles: [], CodecProfiles: [CodecProfile { Type: VideoAudio, Conditions: [ProfileCondition { Condition: Equals, Property: IsSecondaryAudio, Value: "false", IsRequired: False }], ApplyConditions: [], Codec: "aac", Container: null }, CodecProfile { Type: VideoAudio, Conditions: [ProfileCondition { Condition: Equals, Property: IsSecondaryAudio, Value: "false", IsRequired: False }], ApplyConditions: [], Codec: null, Container: null }, CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: NotEquals, Property: IsAnamorphic, Value: "true", IsRequired: False }, ProfileCondition { Condition: EqualsAny, Property: VideoProfile, Value: "high|main|baseline|constrained baseline", IsRequired: False }, ProfileCondition { Condition: LessThanEqual, Property: VideoLevel, Value: "51", IsRequired: False }, ProfileCondition { Condition: NotEquals, Property: IsInterlaced, Value: "true", IsRequired: False }], ApplyConditions: [], Codec: "h264", Container: null }, CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: NotEquals, Property: IsAnamorphic, Value: "true", IsRequired: False }, ProfileCondition { Condition: EqualsAny, Property: VideoProfile, Value: "main", IsRequired: False }, ProfileCondition { Condition: LessThanEqual, Property: VideoLevel, Value: "120", IsRequired: False }, ProfileCondition { Condition: NotEquals, Property: IsInterlaced, Value: "true", IsRequired: False }], ApplyConditions: [], Codec: "hevc", Container: null }], ResponseProfiles: [ResponseProfile { Container: "m4v", AudioCodec: null, VideoCodec: null, Type: Video, OrgPn: null, MimeType: "video/mp4", Conditions: [] }], SubtitleProfiles: [SubtitleProfile { Format: "vtt", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ass", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ssa", Method: External, DidlMode: null, Language: null, Container: null }] }
[2021-03-24 23:43:35.839 +00:00] [ERR] Error in "ffprobe"
System.Exception: ffprobe failed - streams and format are both null.
   at MediaBrowser.MediaEncoding.Encoder.MediaEncoder.GetMediaInfoInternal(String inputPath, String primaryPath, MediaProtocol protocol, Boolean extractChapters, String probeSizeArgument, Boolean isAudio, Nullable`1 videoType, Boolean forceEnableLogging, CancellationToken cancellationToken)
   at MediaBrowser.Providers.MediaInfo.FFProbeVideoInfo.ProbeVideo[T](T item, MetadataRefreshOptions options, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.MetadataService`2.RunCustomProvider(ICustomMetadataProvider`1 provider, TItemType item, String logName, MetadataRefreshOptions options, RefreshResult refreshResult, CancellationToken cancellationToken)
[2021-03-24 23:43:37.019 +00:00] [INF] User policy for "alex". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2021-03-24 23:43:37.051 +00:00] [INF] Profile: "Unknown Profile", Path: "/media/Movies/THE_ACCOUNTANT.iso", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-03-24 23:43:37.074 +00:00] [INF] Profile: "Unknown Profile", No video direct play profiles found for "/media/Movies/THE_ACCOUNTANT.iso" with codec "Unknown codec"
[2021-03-24 23:43:37.126 +00:00] [INF] Profile: "Unknown Profile", Path: "/media/Movies/THE_ACCOUNTANT.iso", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-03-24 23:43:37.127 +00:00] [INF] Profile: "Unknown Profile", No video direct play profiles found for "/media/Movies/THE_ACCOUNTANT.iso" with codec "Unknown codec"
[2021-03-24 23:43:37.199 +00:00] [WRN] Slow HTTP Response from "http://jellyfin.strugee.net/Items/d0f51d98e81f4a862b2f2baa373c2556/PlaybackInfo?UserId=fe460ca06e48491a99fb38da14f1a20e&StartTimeTicks=0&IsPlayback=true&AutoOpenLiveStream=true&MaxStreamingBitrate=140000000" to "10.133.196.1" in 0:00:01.7082024 with Status Code 200
[2021-03-24 23:43:38.183 +00:00] [INF] /usr/lib/jellyfin-ffmpeg/ffmpeg -f iso -i bluray:"/media/Movies/THE_ACCOUNTANT.iso" -map_metadata -1 -map_chapters -1 -threads 0 -sn   -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_base_url "hls/bee06ea14e65ef9b0f374ecd049c225c/" -hls_playlist_type event -hls_segment_filename "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c%d.ts" -y "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c.m3u8"
[2021-03-24 23:43:38.414 +00:00] [ERR] FFmpeg exited with code 1
[2021-03-24 23:43:41.914 +00:00] [INF] Starting Library Sync...
[2021-03-24 23:43:41.918 +00:00] [INF] "LIBRARYSYNC" Added 0 items:  ""
[2021-03-24 23:43:42.062 +00:00] [INF] "LIBRARYSYNC" Updated 1 items:  "d0f51d98e81f4a862b2f2baa373c2556"
[2021-03-24 23:43:42.063 +00:00] [INF] "LIBRARYSYNC" Removed 0 items:  ""
[2021-03-24 23:43:42.066 +00:00] [INF] Finished Library Sync Taking "00:00:00.1498068"
[2021-03-24 23:49:40.269 +00:00] [INF] WS "::ffff:10.133.196.1" closed
```

And the ffmpeg log:

```
/videos/d0f51d98-e81f-4a86-2b2f-2baa373c2556/live.m3u8

{"Protocol":0,"Id":"d0f51d98e81f4a862b2f2baa373c2556","Path":"/media/Movies/THE_ACCOUNTANT.iso","EncoderPath":null,"EncoderProtocol":null,"Type":0,"Container":"iso","Size":null,"Name":"THE_ACCOUNTANT/ISO","IsRemote":false,"ETag":"e236237e1d0282733e349d20c02a7abb","RunTimeTicks":null,"ReadAtNativeFramerate":false,"IgnoreDts":false,"IgnoreIndex":false,"GenPtsInput":false,"SupportsTranscoding":true,"SupportsDirectStream":false,"SupportsDirectPlay":true,"IsInfiniteStream":false,"RequiresOpening":false,"OpenToken":null,"RequiresClosing":false,"LiveStreamId":null,"BufferMs":null,"RequiresLooping":false,"SupportsProbing":true,"VideoType":1,"IsoType":null,"Video3DFormat":null,"MediaStreams":[],"MediaAttachments":[],"Formats":[],"Bitrate":null,"Timestamp":null,"RequiredHttpHeaders":{},"TranscodingUrl":null,"TranscodingSubProtocol":null,"TranscodingContainer":null,"AnalyzeDurationMs":null,"DefaultAudioStreamIndex":null,"DefaultSubtitleStreamIndex":null}

/usr/lib/jellyfin-ffmpeg/ffmpeg -f iso -i bluray:"/media/Movies/THE_ACCOUNTANT.iso" -map_metadata -1 -map_chapters -1 -threads 0 -sn   -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_base_url "hls/bee06ea14e65ef9b0f374ecd049c225c/" -hls_playlist_type event -hls_segment_filename "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c%d.ts" -y "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c.m3u8"


ffmpeg version 4.3.1-Jellyfin Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 8 (Debian 8.3.0-6)
  configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-shared --disable-libxcb --disable-sdl2 --disable-xlib --enable-gpl --enable-version3 --enable-static --enable-libfontconfig --enable-fontconfig --enable-gmp --enable-gnutls --enable-libass --enable-libbluray --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libdav1d --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --arch=amd64 --enable-opencl --enable-vaapi --enable-amf --enable-libmfx --enable-vdpau --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-nvdec --enable-ffnvcodec
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
Unknown input format: 'iso'
```

**Screenshots**

N/A

**Additional context**

```
$ dpkg --get-selections | grep jellyfin
jellyfin                                        install
jellyfin-ffmpeg                                 install
jellyfin-server                                 install
jellyfin-web                                    install

$ cat /etc/apt/sources.list.d/jellyfin.list 
deb [arch=amd64] https://repo.jellyfin.org/debian buster main

$ apt policy 'jellyfin*'
jellyfin-web:
  Installed: 10.7.1-1
  Candidate: 10.7.1-1
  Version table:
 *** 10.7.1-1 500
        500 https://repo.jellyfin.org/debian buster/main amd64 Packages
        100 /var/lib/dpkg/status
jellyfin-ffmpeg:
  Installed: 4.3.1-4-buster
  Candidate: 4.3.1-4-buster
  Version table:
 *** 4.3.1-4-buster 500
        500 https://repo.jellyfin.org/debian buster/main amd64 Packages
        100 /var/lib/dpkg/status
jellyfin-server:
  Installed: 10.7.1-1
  Candidate: 10.7.1-1
  Version table:
 *** 10.7.1-1 500
        500 https://repo.jellyfin.org/debian buster/main amd64 Packages
        100 /var/lib/dpkg/status
jellyfin:
  Installed: 10.7.1-1
  Candidate: 10.7.1-1
  Version table:
 *** 10.7.1-1 500
        500 https://repo.jellyfin.org/debian buster/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by strugee** [comment]
Actually, tinkering with the flags seems to make this work - removing `-f iso bluray:` results in ffmpeg successfully running. So,

    /usr/lib/jellyfin-ffmpeg/ffmpeg "/media/Movies/THE_ACCOUNTANT.iso" -map_metadata -1 -map_chapters -1 -threads 0 -sn   -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_base_url "hls/bee06ea14e65ef9b0f374ecd049c225c/" -hls_playlist_type event -hls_segment_filename "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c%d.ts" -y "/var/lib/jellyfin/transcodes/bee06ea14e65ef9b0f374ecd049c225c.m3u8"

The command hasn't finished running yet so I don't know if this results in a playable movie in the web interface.

**Comment by strugee** [comment]
I didn't realize that Jellyfin would use a different id for each transcode, so running the command manually doesn't work. Therefore I hacked up a workaround to this (run all three commands as root):

```sh
$ dpkg-divert --local --divert /usr/lib/jellyfin-ffmpeg/ffmpeg.real --rename --add /usr/lib/jellyfin-ffmpeg/ffmpeg
Adding 'local diversion of /usr/lib/jellyfin-ffmpeg/ffmpeg to /usr/lib/jellyfin-ffmpeg/ffmpeg.real'

$ cat > /usr/lib/jellyfin-ffmpeg/ffmpeg <<EOF
> #!/bin/bash
> 
> set -eu
> 
> echo $#
> test $# -gt 4 || exec /usr/lib/jellyfin-ffmpeg/ffmpeg.real $@
> 
> new_1="$1"
> new_2="$2"
> 
> if [ $1 = -f ] && [ $2 = iso ] && echo "$4" | grep -q '^bluray:'; then
>         shift
>         shift
>         new_1="$1"
>         new_2="$(echo $2 | sed 's/^bluray://g')"
> fi
> 
> shift
> shift
> 
> exec /usr/lib/jellyfin-ffmpeg/ffmpeg.real $new_1 $new_2 $@
> EOF

$ chmod +x /usr/lib/jellyfin-ffmpeg/ffmpeg
```

This allowed me to "play an ISO" in the web interface - I put that in quotes because actually all that was extracted was the FBI anti-piracy clip. But still! (The actual movie seems to be unplayable due to missing codec support which I'm investigating.)

For those finding this issue looking for a quick fix, this workaround will survive Jellyfin upgrades - but the real way this should be fixed is in Jellyfin's code and you should undo this workaround when that happens. This is basically just a proof-of-concept.

**Comment by strugee** [comment]
> This allowed me to "play an ISO" in the web interface - I put that in quotes because actually all that was extracted was the FBI anti-piracy clip. But still! (The actual movie seems to be unplayable due to missing codec support which I'm investigating.)

This turns out to be incorrect. There's another bug somewhere that makes the timestamp funky, but this actually ended up decoding what appears to be the whole movie.

**Comment by cvium** [comment]
Fixed by #5620

**Comment by strugee** [comment]
Unless I'm missing something, this bug is not fixed by #5620. See https://github.com/jellyfin/jellyfin/pull/5620#issuecomment-808958401. Please reopen.
